.github/workflows/main.yaml

```diff
@@ -36,11 +36,16 @@ jobs:
         uses: actions/setup-node@v3
         with:
           node-version: 18.x
-          cache: 'yarn'
-          cache-dependency-path: ./frontend/yarn.lock
+          cache: "yarn"
+
+      - name: "Install shared module dependencies"
+        run: yarn workspace vaa-shared install --frozen-lockfile
+
+      - name: "Build shared module"
+        run: yarn workspace vaa-shared build
 
       - name: "Install Frontend dependencies"
-        run: yarn install
+        run: yarn install --frozen-lockfile
```

Comment:
That's just a good practice, also helps to avoid unwanted cross-OS changes within the lock file.